include/lib/graft/context.h

```diff
@@ -14,11 +15,11 @@
 #include <chrono>
 #include <any>
 
-#include "graft_utility.hpp"
-#include "graft_constants.h"
+#include "lib/graft/graft_utility.hpp"
+#include "lib/graft/graft_constants.h"
 
 namespace graft { class ConfigOpts; }
-namespace graft::supernode::system_info { class Counter; }
+namespace graft::request::system_info { class Counter; }
```

Comment:
that perhaps should be graft::system_info

@cybay or @AlexanderSuprunenko could you fix that please?